client/my-sites/email/email-management/home/email-list-inactive.jsx

```diff
@@ -1,18 +1,50 @@
 import { Button, Card } from '@automattic/components';
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
+import PromoCard from 'calypso/components/promo-section/promo-card';
 import SectionHeader from 'calypso/components/section-header';
 import { canCurrentUserAddEmail } from 'calypso/lib/domains';
-import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+import {
+	emailManagementPurchaseNewEmailAccount,
+	isUnderEmailManagementInbox,
+} from 'calypso/my-sites/email/paths';
 
 class EmailListInactive extends React.Component {
+	getMainHeader() {
+		const { translate } = this.props;
+		const image = {
+			path: emailIllustration,
+			align: 'right',
+		};
+
+		return (
+			<>
+				<PromoCard title={ translate( 'Pick a domain to get started' ) } image={ image }>
+					<p>
+						{ translate(
+							'Pick a domain from your available domains below to add an email solution.'
+						) }
+					</p>
+				</PromoCard>
+				<br />
+			</>
+		);
+	}
+
 	render() {
 		const { selectedSiteSlug, currentRoute, domains, translate } = this.props;
 
 		if ( domains.length < 1 ) {
 			return null;
 		}
 
+		const sectionHeaderLabel = isUnderEmailManagementInbox( currentRoute )
+			? translate( 'Domains' )
+			: translate( 'Other domains' );
+
+		const mainHeader = isUnderEmailManagementInbox( currentRoute ) ? this.getMainHeader() : null;
+
```

Comment:
I would prefer it if this logic was handled by props, where we can push this logic onto the caller - it doesn't feel right to me for this component to need to know what context it is running in.
```suggestion
		const { currentRoute, domains, header, sectionLabel, selectedSiteSlug, translate } = this.props;

		if ( domains.length < 1 ) {
			return null;
		}

		const sectionHeaderLabel = sectionLabel ? sectionLabel : translate( 'Other domains' );

```

Reply:
I have reworked this but added modifications in the caller (email-home) to pass those properties to the child (email-list-inactive).

Flow is that inbox-management calls email-home with parameters and email-home calls email-list-inactive with those parameters.